invenio_theme/assets/semantic-ui/less/invenio_theme/theme/elements/segment.overrides

```diff
@@ -11,3 +11,12 @@
     color: black;
     padding: 0.7em;
 }
+
+/* Brand */
+.ui.brand.segment:not(.inverted) {
+  border-top: @coloredBorderSize solid @brandColor !important;
+}
+.ui.inverted.brand.segment {
+  background-color: @brandColor !important;
+  color: @white !important;
+}
```

Comment:
Following the example of defining colors in SUI https://github.com/Semantic-Org/Semantic-UI/blob/master/src/definitions/elements/segment.less#L550-L557, you can now define a new color for the segment in the format `ui brand segment` or `ui brand inverted segment`.